test/TwilioVoiceReactNativeExample/e2e/basic.test.ts

```diff
@@ -0,0 +1,120 @@
+import { device, element, expect as detoxExpect, by, waitFor } from 'detox';
+import { expect as jestExpect } from 'expect';
+import twilio from 'twilio';
+
+const DEFAULT_TIMEOUT = 10000;
+
+const bootstrap = () => {
+  const accountSid = process.env.ACCOUNT_SID;
+  const authToken = process.env.AUTH_TOKEN;
+  const mockClientId = process.env.CLIENT_IDENTITY;
+
+  if (
+    [accountSid, authToken, mockClientId].some((v) => typeof v !== 'string')
+  ) {
+    throw new Error('Missing env var.');
+  }
+
+  const twilioClient = twilio(accountSid, authToken);
+
+  return { twilioClient, clientId: mockClientId as string };
+};
+
+describe('basic', () => {
+  let twilioClient: ReturnType<typeof twilio>;
+  let clientId: string;
+
+  const doRegister = async () => {
+    await element(by.text('REGISTER')).tap();
+    await waitFor(element(by.text('Registered: true')))
+      .toBeVisible()
+      .withTimeout(DEFAULT_TIMEOUT);
+  };
+
+  beforeAll(async () => {
+    ({ twilioClient, clientId } = bootstrap());
+
+    await device.launchApp();
+  });
+
+  beforeEach(async () => {
+    await device.reloadReactNative();
+  });
+
+  it('should show the SDK version', async () => {
+    const el = element(by.id('sdk_version'));
+    await detoxExpect(el).toBeVisible();
+    const sdkVersionAttr = await el.getAttributes();
+    if (!('text' in sdkVersionAttr)) {
+      throw new Error('could not parse text of sdk version element');
+    }
+    const sdkVersionText = sdkVersionAttr.text;
+    jestExpect(sdkVersionText).toMatch('SDK Version: ');
+    jestExpect(sdkVersionText).not.toMatch('SDK Version: unknown');
```

Comment:
Does detox matcher not work here?

Reply:
I wanted to perform regex on the text contents of the element.